apps/api/src/domain/models/audio-visual/audio-item/commands/translate-audio-item-name/audio-item-name-translated.event-handler.ts

```diff
@@ -0,0 +1,29 @@
+import { MultilingualTextItemRole } from '@coscrad/api-interfaces';
```

Comment:
Did you register this handler in the `AudioItemModule`? If not, it won't be handled when the corresponding command publishes the event.

Reply:
This still needs to be done.

Reply:
I see this has been done (in the `AudioVisualModule`) now.